protocol/0072-SPPW-spam-protection-PoW.md

```diff
@@ -74,5 +74,46 @@ The initial hash-function used is SHA3 . To allow for a more fine-grained contro
 - Linking too many transactions to the same block is detected and leads to a blocking of that account (if the increasing difficulty is turned of) (<a name="0072-SPPW-003" href="#0072-SPPW-003">0072-SPPW-003</a>)
 - Linking too many transactions with a low difficulty level to a block is detected and leads to blocking of the account (if increasing difficulty is turned on) (<a name="0072-SPPW-004" href="#0072-SPPW-004">0072-SPPW-004</a>)
 - Reusing a transaction identifier in a way that several transactions with the same ID end up in the same block is detected and the transactions are rejected (<a name="0072-SPPW-005" href="#0072-SPPW-005">0072-SPPW-005</a>)
-- A blocked account is unblocked after 4 epochs. (<a name="0072-SPPW-006" href="#0072-SPPW-006">0072-SPPW-006</a>)
+- A blocked account is unblocked after 5 minutes. For transactions sent in the meantime, a meaningfull error message is returned. (<a name="0072-SPPW-006" href="#0072-SPPW-006">0072-SPPW-006</a>)
```

Comment:
Isn't that 1/48 of an epoch now and already in through Tom's cleanup (i.e., should be removed here) ?

Reply:
@Vegaklaus

@claudiumilea and I noticed that in the main branch this still shows as 4 epochs. Ive created https://github.com/vegaprotocol/specs/pull/1480 - I can close this if this PR updates and corrects if its no longer 4 epochs.